Update instructions for setting up AtomSampleViewer. Remove AtomTest … (#1594)
o3de/o3de.org: content/docs/atom-guide/setup-atom-projects.md

---
linkTitle: "Setup"
title: "Setting up Atom Renderer"
description: "Get started with Atom Renderer with an Open 3D Engine project or AtomSampleViewer, Atom's standalone application."
toc: true
weight: 200
---

Get started with **Atom Renderer** through **Open 3D Engine (O3DE)** by creating a new *O3DE project* or building **Atom Sample Viewer**, Atom's standalone application. 

## Prerequisites

Before setting up a new O3DE project or Atom Sample Viewer, you must install O3DE on your computer. Follow the instructions in [Setting Up O3DE](/docs/welcome-guide/setup) of the Get Started Guide.

> ## O3DE Project

Reviewer: ```suggestion
## O3DE project
```
For section headers, use sentence casing. To that end, changed "Project" to "project".

> Creating an O3DE project provides you the full suite of tools, including **O3DE Editor**, **Material Editor**, and a collection of Gems essential to building a simulation or game. This is a great way to test out Atom's features among other objects and mechanics in a level. 

Reviewer: ```suggestion
Creating an O3DE project provides you the full suite of O3DE tools, including **O3DE Editor**, **Material Editor**, and a collection of Gems essential to building a simulation or game. This is a great way to test out Atom's features among other objects and mechanics in a level.
```
For clarity, changed "the full suite of tools" to "the full suite of O3DE tools".

![Default level prefab](/images/learning-guide/tutorials/environments/create-a-level-C.png)

### Setup 

> 1. Follow the instructions in[Creating a project](/docs/welcome-guide/create/) of the Get Started guide. Atom is built into O3DE via the **Atom Gem**, a core Gem that's enabled in your project by default.

Reviewer: ```suggestion
1. Follow the instructions in[Project Creation](/docs/welcome-guide/create/) in the Get Started guide. After creating an O3DE project, you have access to Atom via the **Atom Gem**. This is a core Gem that's built into O3DE and enabled in your project by default.
```
* _Same cross-reference link changes as for line 13_
* For a clearer throughline from the "follow the instructions" sentence to the sentence about the Atom Gem, I changed "Atom is built into O3DE" to the expanded "After creating an O3DE project, you have access to Atom". (I also split the latter sentence for flow.) I thought that this was clearer, but see what you think.

> 2. After you set up your project, check out the following pages to explore Atom's features via the O3DE Editor:

Reviewer: ```suggestion
1. After you set up your project, check out the following pages to explore Atom's features via O3DE Editor:
```
* Per our [formatting guidelines for ordered lists](https://docs.o3de.org/docs/contributing/to-docs/style-guide/format/#ordered-lists), use "1." for all ordered list items.
* Don't use "the" when referring to certain proper noun tool names, such as O3DE Editor.

   - [Create a Level](/docs/learning-guide/tutorials/environments/create-a-level/): An introduction to your first level. Whenever you create a level in your project, the default environment consists of many Atom components. 

>    - [Component Reference - Atom](/docs/user-guide/components/reference/#atom): The full list of Atom components provided by the Atom Gem. 

Reviewer: ```suggestion
   - [Component Reference -- Atom](/docs/user-guide/components/reference/#atom): The full list of Atom components that the Atom Gem provides. 
```

## Atom Sample Viewer

> Atom Sample Viewer is a good place to start if you're interested in evaluating Atom's features in a controlled environment. It's a standalone application, meaning you don't need to open it with O3DE Editor. Rather, it showcases samples that demonstrate each feature individually and with parameters you can tweak and configure. 

Reviewer 2: This doesn't read to me like a contrasting statement, so I think "Rather" is unnecessary.

```suggestion
Atom Sample Viewer is a good place to start if you're interested in evaluating Atom's features in a controlled environment. It's a standalone application, so you don't need to open it with O3DE Editor. It showcases samples that demonstrate each feature individually with parameters you can tweak and configure. 
```

> Check out the [`o3de-atom-sampleviewer`](https://github.com/o3de/o3de-atom-sampleviewer) repository in GitHub.

Reviewer: ```suggestion
To use the Atom Sample Viewer, get a local copy from the [`o3de-atom-sampleviewer`](https://github.com/o3de/o3de-atom-sampleviewer) repository in GitHub.
```
Added some additional context for the GitHub link.

Author: I decided to just remove this line because I didn't want to give the user an action item at this point in the doc. Especially since immediately after are instructions to get the repository.

> ![An example of Atom Sample Viewer, a standalone application. This image showcases the Shadow sample.](/images/atom-guide/atom-sample-viewer/shadow.png)

Reviewer: ```suggestion
![An image of the Shadow sample in the Atom Sample Viewer.](/images/atom-guide/atom-sample-viewer/shadow.png)
```
Suggestion to condense the wording here. I don't think that you need to mention again that Atom Sample Viewer is "a standalone application"–that's already established in the topic intro.

### Setup

1. In a command line window, clone the [`o3de-atom-sampleviewer`](https://github.com/o3de/o3de-atom-sampleviewer) repository onto your local machine.

    ```cmd
    git clone https://github.com/o3de/o3de-atom-sampleviewer.git
    ```

> 2. Update your local repository by pulling the the latest changes, if any.

Reviewer: ```suggestion
1. Update your local repository by pulling the the latest changes, if any.
```

    ```cmd    
    git pull origin main
    ```

> 3. Use CMake to generate a Visual Studio solution, which is used in the next step to build the Atom project. 

Reviewer: ```suggestion
1. Use CMake to generate a Visual Studio solution. In the next step, you build the Atom project using this solution.
```
For flow and to avoid passive voice ("is used"), revised the second half of this sentence and split it into its own sentence.

    ```cmd
    cmake -B build/windows_vs2019 -S . -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:/o3de-packages
    ```

> 4. Use CMake to build Atom Sample Viewer Standalone. Specify the `profile` configuration for your build.

Reviewer: ```suggestion
1. Use CMake to build `AtomSampleViewerStandalone`. Specify the `profile` configuration for your build.
```
